app/controllers/concerns/course/lesson_plan/strategies/fomo_personalization_strategy.rb

```diff
@@ -82,9 +82,9 @@ def cannot_shift_item(course_user, item, submitted_items, items_to_shift)
 
   def item_is_straggling(personal_time, reference_time)
     if reference_time.end_at.present? && personal_time.end_at.present?
-      reference_time.end_at < personal_time.end_at
+      return reference_time.end_at < personal_time.end_at
     elsif reference_time.end_at.present? && personal_time.end_at.nil?
-      true
+      return true
     end
```

Comment:
might be better to use `if`-`elsif`-`else`-`end` to align with the rest of the codebase.